index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>Tumworobere Annah</title>
+</head>
+
+<body>
+    <header class="site-header">
+
+        <a href="#"> Tumworobere Annah</a>
+        <nav class="nav-menu">
+            <img src="./images/Vector.png" alt="Vector image ">
+        </nav>
+
+    </header>
+    <main>
+        <section class="container">
+
+            <div class="text">
+                <h1> Hey There. I'm  Annah</h1>
+
+
+                <h3>I'm a Software Developer.</h3>
+
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+
+            </div>
+            <div class="icon">
+                <ul>
+                    <li><a href="#"><img src="./images/img1.png" alt="Github-icon"></a></li>
+                    <li><a href="#"><img src="./images/img2.png" alt="Linkedin-icon"></a></li>
+                    <li><a href="#"><img src="./images/img3.png" alt="Twitter-icon"></a></li>
+                    <li><a href="#"><img src="./images/img4.png" alt="AngelList-icon"></a></li>
+                    <li><a href="#"><img src="./images/img5.png" alt="Medium-icon"></a></li>
+
+                </ul>
+            </div>
+
+
+
+        </section>
+    
+        
+    
+        
```

Comment:
- [ ] Please remove unnecessary blank lines as provided in the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). This makes for code readability and organization. Also, do this for other parts of your code.

![blank_m](https://user-images.githubusercontent.com/36542986/134498082-19c9adbb-cf99-48a2-a8e0-630581a515b4.jpg)